### ISSUE TITLE
Duplicate keys generated

### ISSUE THREAD

**Issue by coodoo** [noffle/bisecting-numbers#1]
This seems to be an interesting bug, see sample code below, it won't happen with Dominic's `between`, any idea why?

``` javascript
var between = require('bisecting-between')()
// var between = require('between')
between( between.lo, between.hi ) // 1
between( between.lo, '1' ) // -1
between( between.lo, '-1' ) // -1 ← this should be a different key instead of -1 again!
```

**Comment by hackergrrl** [comment]
Did you mean to file this under [bisecting-between](https://github.com/noffle/bisecting-between) instead?

This is fixed as of [0.0.8](https://github.com/noffle/bisecting-between/commits/master). Thanks!

**Comment by coodoo** [comment]
Yep it should be issued there, good job fixing it, thanks!
